### PR TITLE
Added author info

### PR DESCRIPTION
**Pull request by bansalvi**
### Checklist

- [ ] Text is formatted so that each sentence is on its own line.

- [ ] Pre-prints cited in this pull request have a GitHub issue opened so that they can be reviewed.

### CODE REVIEW

File: content/metadata.yaml [hunk 1]
```diff
@@ -504,4 +504,22 @@ authors:
       string: "None"
       lastapproved: !!str 2020-04-30
     funders: NHGRI R01 HG10067
-
+
```
Comment (rando2) [inline content/metadata.yaml:507]:
```suggestion
```

File: content/metadata.yaml [hunk 2]
```diff
@@ -504,4 +504,22 @@ authors:
...
+    github: bansalvi
+    name: Vikas Bansal
+    initials: VB
+    orcid: 0000-0002-0944-7226
+    twitter: VikasBansal1989
+    email: bansal.bioinfo@gmail.com
+    contributions:
+      - Writing - Original Draft
+      - Investigation
+      - Visualization
```
Comment (rando2) [inline content/metadata.yaml:518]:
```suggestion
```
We are trying to keep it up to date with what we've already added into the document, so just we just need to remember to add this back in when we add in your visualization!

**Comment by bansalvi** [comment]
@rando2 Thanks. Make sense. I was not really sure about that. Also, was wondering if I reviewed a paper, should I add directly to https://github.com/greenelab/covid19-review/blob/master/content/95-ismms-himc-appendix.md
Actually already tried to include one review :)

**Comment by cgreene** [comment]
@cornhundred : would you take a contributed review upstream at https://github.com/ismms-himc/covid-19_sinai_reviews/ ?

**Comment by rando2** [comment]
@bansalvi I am going to merge this, but @cornhundred should still be able to answer here! You can also add your review to the Issues (just search for the title or DOI of a paper you reviewed, and if it isn't an open issue yet, just make a new one!)

**Pull request merged**